### DISCUSSION TITLE
jvm/what-is-jvm

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#89] [Announcements]
# jvm/what-is-jvm

Java程序员进阶之路，小白的零基础Java教程，从入门到进阶，JVM到底是什么？

https://tobebetterjavaer.com/jvm/what-is-jvm.html

**Comment by CofJus** [comment]
写了好几年Java了，头回看到这么清晰易懂的文章666

**Comment by fantasy-zrq** [comment]
二哥，实在是太清晰了！
